zubhub_frontend/zubhub/package.json

```diff
@@ -6,6 +6,7 @@
     "@date-io/date-fns": "^1.3.13",
     "@material-ui/core": "^4.11.2",
     "@material-ui/icons": "^4.11.2",
+    "@material-ui/lab": "^4.0.0-alpha.61",
```

Comment:
You can probably just revert the package.json file to what it is on master because there shouldn't be any changes here.